tools/scripts/upgrades/increase-block-size.sh

```diff
@@ -0,0 +1,20 @@
+echo "Current params:"
+poktrolld query consensus params
+
+echo "Submitting transaction to change the block size..."
+poktrolld tx authz exec tools/scripts/params/consensus_increase_block_size.json --from pnf --yes
+
+sleep 3
+echo "New params:"
+poktrolld query consensus params
+
+
+# DISCUSS_IN_THIS_PR: do we need to add authorization to change staking params? Consider other modules as well.
```

Comment:
@Olshansk Should we enable changes to other core module parameters? The main concern is that DAO/key holders might have too much power over key parameters. Assuming all commands are thorough tested on TestNet before executing against MainNet.

We could potentially extend the ability to change parameters beyond just block size, proof, and claims to all core module params.

Reply:
Just discussed this on a call. DAO is a [GM](https://en.wikipedia.org/wiki/Gamemaster), and we're going to add more authorizations in a separate PR to include the authz authorizations in the next release.

Reply:
![maxresdefault](https://github.com/user-attachments/assets/4be5d842-78ef-4fc5-b010-107572dfbae4)